docs/Code_commit_process/README.md

```diff
@@ -1,3 +1,35 @@
 # Test prerequisite for Code Commit
 
 ![Dent-SIT](../img/code_commit.png)
+
+To run the pre-commit checks locally, you can follow below steps:
+
+- Ensure that default python is python3.
+
+- Ensure that the `pre-commit` package is installed:
+
+```Shell
```

Comment:
Indent it by four spaces, so it’s mapped(?) to the item?

Reply:
done